ci/jenkinsfiles/release.groovy

```diff
@@ -0,0 +1,335 @@
+])
+
+String currentVersion() {
+  return readMavenPom().getVersion()
+}
+
+String getReleaseVersion(String version) {
+  return version.replace('-SNAPSHOT', '')
+}
+
+pipeline {
+  agent {
+    label 'builder-maven-nuxeo-11'
+  }
+  parameters {
+    string(name: 'rcVersion', description: 'Version to be promoted')
+    string(name: 'reference', description: 'Reference branch to be bumped after releasing')
+    booleanParam(
+      name: 'dryRun', defaultValue: true,
+      description: 'if true all steps will be run without publishing the artifact'
+    )
+  }
+  environment {
+    APP_NAME = "${appName}"
+    BACKEND_FOLDER = "${WORKSPACE}/nuxeo-coldstorage"
+    BRANCH_NAME = GIT_BRANCH.replace('origin/', '')
+    BRANCH_LC = "${BRANCH_NAME.toLowerCase().replace('.', '-')}"
+    CONNECT_PROD_URL = 'https://connect.nuxeo.com/nuxeo'
+    CHART_DIR = 'ci/helm/preview'
+    CURRENT_VERSION = currentVersion()
+    ENABLE_GITHUB_STATUS = 'false'
+    FRONTEND_FOLDER = "${WORKSPACE}/nuxeo-coldstorage-web"
+    JENKINS_HOME = '/root'
+    MAVEN_DEBUG = '-e'
+    MAVEN_OPTS = "${MAVEN_OPTS} -Xms512m -Xmx3072m"
+    ORG = 'nuxeo'
+    VERSION = getReleaseVersion(CURRENT_VERSION)
+  }
+  stages {
+    stage('Check parameters') {
+      steps {
+        script {
+          if (!params.rcVersion || params.rcVersion == '') {
+            currentBuild.result = 'ABORTED'
+            currentBuild.description = 'Missing required version parameter, aborting the build.'
+            error(currentBuild.description)
+          }
+
+          if (!params.reference || params.reference == '') {
+            currentBuild.result = 'ABORTED'
+            currentBuild.description = 'Missing required reference parameter, aborting the build.'
+            error(currentBuild.description)
+          } else {
+            echo '''
+              ----------------------------------------
+              Update Reference Branch
+              ----------------------------------------
+            '''
+            env.REFERENCE_BRANCH = params.reference
+          }
+
+          if (params.dryRun && params.dryRun != '') {
+            env.DRY_RUN_RELEASE = params.dryRun
+            if (env.DRY_RUN_RELEASE == 'true') {
+              env.SLACK_CHANNEL = 'infra-napps'
+            }
+          } else {
+            env.SLACK_CHANNEL = 'pr-napps'
+            env.DRY_RUN_RELEASE = 'false'
+          }
+          env.RC_VERSION = params.rcVersion
+          env.PACKAGE_BASE_NAME = "nuxeo-coldstorage-package-${VERSION}"
+          env.PACKAGE_FILENAME = "nuxeo-coldstorage-package/target/${PACKAGE_BASE_NAME}.zip"
+          echo """
+            -----------------------------------------------------------
+            Release nuxeo coldstorage connector
+            -----------------------------------------------------------
+            ----------------------------------------
+            Coldstorage package:   ${PACKAGE_BASE_NAME}
+            Build version:    ${RC_VERSION}
```

Comment:
for `RC_VERSION` I don't know if we should keep the same naming as know we don't have a real `rc` tag but a build number, any way I think we can keep as it's. Even the build number is a release candidate. we can check what we did on Platform.

Reply:
They call it 'BUILD_VERSION'

Reply:
@doulba I think we will update it but not for now. Please see my comment https://github.com/nuxeo/nuxeo-coldstorage/pull/12#discussion_r543405838

Reply:
@doulba with `BUILD_VERSION` do we need to change something on our Jenkins job or not? I mean the release one where we provide the `rc` / build number to promote.

Reply:
No needed, we will only have to provide the build version ( or release candidate, it's just a name).
for example:
if I want to promote the Jenkins Build 2021.0.1 as final release the BUILD_VERSION parameter will be equal to 2021.0.1

Reply:
is what I had in mind, thx for confirmation